tests/testapp/home/static/button_entity.js

```diff
@@ -0,0 +1,6 @@
+const ButtonDecorator = ({ entityKey, children }) => {
+  const attrs = {'data-tooltip': entityKey, className: 'RichEditor-button'};
+  return window.wagtaildraftail.createElement('span', attrs, children);
+};
```

Comment:
Honestly I'm not sure I want to encourage people to do this without a build step. It's fine for this PR but I might add one later on.